### PR TITLE
LRQA-66039 | Investigate failure in Theme#CanBeAppliedToPageWhenClassicThemeNotAvailable

### PR DESCRIPTION
**Pull request by alexmds2004**
**Background**:
Fix "failed with welcome portlet"

**Test Plan:**
"Verify portal will apply different available page theme if classic theme not available"

**JIRA Ticket:**
https://issues.liferay.com/browse/LRQA-66039?jql=ORDER%20BY%20lastViewed%20DESC

### CODE REVIEW

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase [hunk 1]
```diff
@@ -8,6 +8,8 @@ definition {
 
 	setUp {
 		TestCase.setUpPortalInstance();
+
+		User.firstLoginPG();
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase:12]:
```suggestion

```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase [hunk 2]
```diff
@@ -29,8 +31,6 @@ definition {
 	test CanBeAppliedToPage {
 		property test.name.skip.portal.instance = "Theme#CanBeAppliedToPage";
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase:33]:
```suggestion
                   User.firstLoginPG();
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase [hunk 3]
```diff
@@ -93,8 +93,6 @@ definition {
 
 		var appName = "test-theme";
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase:95]:
```suggestion
                      User.firstLoginPG();
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase [hunk 4]
```diff
@@ -168,8 +166,6 @@ definition {
...
```
Comment (mateusralv) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/frontendinfrastructure/themesinfrastructure/Theme.testcase:168]:
```suggestion
                       User.firstLoginPG();
```

**Review by mateusralv — CHANGES_REQUESTED** [review]
Hey @alexmds2004 could you

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by alexmds2004** [comment]
ci:test:sf

**Comment by alexmds2004** [comment]
ci:test:frontend

**Comment by mateusralv** [comment]
Closed for further investigation.

See: https://liferay.slack.com/archives/C01FP01V5L0/p1680293941982119

**Pull request closed without merging**